docs/spring-boot-runtime/master.adoc

```diff
@@ -14,6 +14,20 @@ include::topics/sb-tomcat-runtime-overview.adoc[leveloffset=+1]
 
 {SpringBoot} has a version of its runtime that is tested and verified to run on OpenShift. Some components, for example, the Embedded Tomcat Container and the Java container image, are provided as part of a Red Hat subscription when used with {SpringBoot}.
 
+The features below are available as a collection of missions and quickstarts link:{link-launcher-oso}[ on {OpenShiftOnline}] or your {OpenShiftLocal}.
```

Comment:
Remove `and quickstarts`, remove double space, and reword:

> The features in the below table are available as a collection of missions which run on OpenShift.

Reply:
Thanks @rhoads-zach @metacosm  can you  take a look. I'm sending you a link to the html /pdf of the guide as well.